### PR TITLE
[bug 1137186] Start asking question message

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: app/question.html [hunk 1]
```diff
@@ -38,9 +38,8 @@
         <h1>My Question</h1>
       </header>
       <main id="thread-scroll" class="fit scroll">
-        <section id="thread-introduction" class="thread-introduction">
```
Comment (Rik) [inline app/question.html]: We need to remove the CSS in buddyup.css that was using this class.

File: app/css/question.css [hunk 2]
```diff
@@ -7,3 +7,27 @@
...
+    height: 100%;
+    justify-content: center;
+    text-align: center;
+}
+.StartAskingMessage > span {
+    padding-top: 11.4rem;
+    position: relative;
+}
+.StartAskingMessage > span::before {
+    background: url('ui-images/img_start_asking@2x.png');
```
Comment (Rik) [inline app/css/question.css]: Move the image in `app/css/ui-images/`

Comment (Rik) [inline app/css/question.css]: I've tried a simpler solution with less "magic values" and repetitions. Here it is in a diff:
```diff
 app/css/question.css | 12 ++++--------
 app/question.html    |  2 +-
 2 files changed, 5 insertions(+), 9 deletions(-)

diff --git a/app/css/question.css b/app/css/question.css
index 6107f3f..43b7840 100644
--- a/app/css/question.css
+++ b/app/css/question.css
@@ -10,24 +10,20 @@
 .StartAskingMessage {
     align-items: center;
     color: #4d4d4d;
-    display: flex;
     font-size: 1.6rem;
     height: 100%;
     justify-content: center;
     text-align: center;
 }
 .StartAskingMessage > span {
-    padding-top: 11.4rem;
-    position: relative;
+    display: block;
 }
-.StartAskingMessage > span::before {
+.StartAskingMessage::before {
     background: url('ui-images/img_start_asking@2x.png');
     background-size: cover;
     content: '';
+    display: block;
     height: 9.6rem;
-    left: 50%;
-    margin-left: -4.8rem;
-    position: absolute;
-    top: 0;
     width: 9.6rem;
+    margin-bottom: 18px;
 }
diff --git a/app/question.html b/app/question.html
index b48eb2d..b4eaeb8 100644
--- a/app/question.html
+++ b/app/question.html
@@ -38,7 +38,7 @@
         <h1>My Question</h1>
       </header>
       <main id="thread-scroll" class="fit scroll">
-        <section id="thread-introduction" class="StartAskingMessage">
+        <section id="thread-introduction" class="StartAskingMessage vbox">
           <span>We can help! Ask us anything.</span>
         </section>
         <section id="suggestions" class="hide">
```

Comment (rehandalal) [inline app/css/question.css]: the image is already in `app/css/ui-images/`

Comment (Rik) [inline app/css/question.css]: Ah sorry :)

**Pull request closed without merging**